Makefile

```diff
@@ -54,7 +54,10 @@ RM_F = $(RM) -f
 RM_RF = $(RM) -rf
 
 RUNTIME ?= dotnet
-VERSION = $(shell git name-rev --name-only --tags --no-undefined HEAD 2>/dev/null || echo git-`git rev-parse --short HEAD`)
+# Do not use := for variable declaration because the value would be outdated after running target version:
+override CURRENT_VERSION = $(shell head -n1 VERSION)
```

Comment:
I wasn't aware of `override`, can you comment on its portability across bsd etc?

Since this is only used once in `install-linux-shortcuts`, wouldn't it be safer to keep that were it was and avoid the risk?

Reply:
> I wasn't aware of `override`, can you comment on its portability across bsd etc?

I don't know. What version of **make** do you mean exactly? **bmake** (used by FreeBSD) doesn't support it but the Makefile is incompatible with it anyway. I don't know what is used on MacOS.

> Wouldn't it be cleaner to keep that inline and avoid the new variable completely?

Yes, that's the simplest solution.

Reply:
Done, I've remove `CURRENT_VERSION`.

Out of curiosity, why is `?=` used for some variables (e.g. `RUNTIME ?= dotnet`)?

Reply:
To be honest, it probably comes down to confusion over how the different assignment operators work. I would have thought from reading the docs `?=` would be necessary for anything that could be overriden from the terminal, but in practice that has always worked fine with just `=`.